java/offline-geocode/src/main/java/com/esri/arcgisruntime/sample/offlinegeocode/MainActivity.java

```diff
@@ -224,7 +224,6 @@ public void run() {
     mPinSourceSymbol.setWidth(20);
     mPinSourceSymbol.loadAsync();
     mPinSourceSymbol.setLeaderOffsetY(45);
-    mPinSourceSymbol.setOffsetY(-48);
```

Comment:
This was not part of the issue, but I think we should probably fix it anyways...

This code is showing a deprecated way of creating a picture marker symbol (Line 222). Could you change it to look something like: https://github.com/Esri/arcgis-runtime-samples-android/blob/65533272642bd366e31af709c250063ded30219a/java/find-route/src/main/java/com/esri/arcgisruntime/sample/findroute/MainActivity.java#L235

Also the line    `  //[DocRef: Name=Picture Marker Symbol Drawable-android, Category=Fundamentals, Topic=Symbols and Renderers]` is a copy paste error and can be removed. FYI - its a tag for a script we run which pulls out code snippets used in our documentation.